apps/web/src/components/CopyToClipboard.tsx

```diff
@@ -0,0 +1,40 @@
+import { useState } from "react";
+import { CheckIcon } from "@heroicons/react/24/outline";
+
+import Copy from "~/icons/copy.svg";
+import { Tooltip } from "./Tooltip";
+
+export function CopyToClipboard({
+  label,
+  value,
+}: {
+  label: string;
```

Comment:
Let's make `label` optional and default to "Copy to clipboard"

Reply:
Done [46c3b1b](https://github.com/Blobscan/blobscan/pull/478/commits/46c3b1be7d71c10461c6ff60be4c65bf5e6ab071)